tests/test_game_restore_relocation.sh

```
@@ -0,0 +1,78 @@
+
+    local new_save_path="${HOME}/.local/share/reloc-game/new-saves"
+    mkdir -p "${new_save_path}"
+
+    cat > "${XDG_CONFIG_HOME}/instant/games/installations.toml" <<EOF
+[[installations]]
+game_name = "${game_name}"
+save_path = "${new_save_path}"
+nearest_checkpoint = "${snapshot_id}"
+EOF
+
+    rm -rf "${new_save_path}"/*
```

Shellcheck (0.11.0)

[warning] Use "${var:?}" to ensure this never expands to /* .

(SC2115)